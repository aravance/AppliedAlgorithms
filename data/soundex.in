above
accommodable
ackey
adalat
adnexal
aid
aide
airplane
allocate
almost
altercate
amphitheater
anaesthesiologist
anesthesiologist
angel
angelic
anthem
antierosion
antiquarism
aperulosid
append
appendix
areel
arnut
arylated
assault
atame
atweel
autoclaves
avitaminosis
babboon
babboons
bagass
balance
balloon
balloons
banana
basher
bayardly
bedbugs
beice
benadryl
bename
bench
benches
benchmark
bend
bezan
bibber
bilaminated
bird
black
blockhead
blue
boatheader
bob
bobber
bobbie
bobbing
bobby
bobs
bomb
bomber
bop
borofluorin
bow
boy
brand-new
briar
bribe
bromates
bullets
burrowing
bytownite
caissons
calvin
came
candle
cap
capped
cappuccino
cardstock
cartiest
cataracts
cave
caveman
cereal
channels
cheap
cheapskates
cheat
cheater
check
checker
checkerboard
chemical
chemotic
cheque
chequer
chevron
choreographers
circularized
clack
clap
click
client
clientele
clubfist
coffee
college
come
comely
complexity
complicated
condite
connect
context
cooped
cordite
corespondent
counterbend
covariation
crimson
curb
cyanide
cytosin
dampen
dance
dancer
deck
decrier
degentilize
dematiaceous
dentalize
deque
dermatographism
desynonymization
dezinkify
dicaryophase
dikelet
diplomatique
disbelief
disconnected
disleafing
dispel
dissolve
doberman
docile
doom
dosage
double
dragon
drip
drizzle
drogue
dyewood
eburnification
education
elaeometer
elephant
embowerment
enantiotropic
endothermic
enripen
enwing
episcopality
ergotine
evangelise
evangelize
excite
expell
expelled
extrapolate
far
farther
father
feodary
fibrointestinal
finagle
finagling
fission
flavor
florimanist
fog
for
fore
form
formula
four
foy
frog
froth
furnish
fusion
game
gasometrically
geek
geogony
girl
gitanos
glood
gobiesociform
goodby
grade
gravel
grill
grub
gunner
habitan
habitat
hail
hard
harden
hardly
head
heal
hear
heel
here
high
hissing
horse
hub
hypobole
ill
ilk
indicts
indigo
intramuscularly
isothermic
item
jaw
jaws
jillflirt
jovial
jut
karat
karate
karma
kickboard
kishkas
kobird
kryptonite
laciniose
laminate
laps
lattice
lean
lemma
leucocyte
liederkranz
limos
lisped
lobscouse
longirostrine
louvred
lulu
macula
maimer
malthite
mannitols
mar
masculinism
maudle
mean
meanless
mein
menilite
Meservey
meter
micraco
micro
middle
might
milter
mind
mine
miryachit
misjudgment
mississippi
mite
mojarra
monochromate
moonack
music
musical
nasty
nautically
nest
nesting
nestle
nestor
nexum
ninety
nomographic
noncaustically
nondefinable
nonexertive
noninflationary
nonofficinal
nonrectangular
nonsubjectification
nook
novel
obtuse
oncology
ophthalmologist
orthogonal
ostrich
outfigure
outspell
overboard
oxen
pack
page
palgat
panhandler
paregoric
patronize
peck
pecking
persuasion
phalange
phalanx
philosophy
photogenetic
pigeon
pipe
placeable
platter
plough
polo
pop
pope
port
preen
previous
probate
probating
profuse
property
prussia
puccini
pupil
quack
quadrangle
que
queue
quick
quicken
quit
rabid
raccoon
rain
rare
reach
reachable
reachableness
red
refuge
reign
remorse
resell
reunite
rheometric
ricochets
right
rite
road
rob
robber
robbers
robbie
robby
robot
robots
robs
rope
rose
rosetta
rotate
rotates
rotating
rotation
rotations
runner
sack
sacred
sand
sargo
sargon
say
scar
scared
scary
science
scientific
scientist
scientists
scissor
screw
sea
searcy
see
seen
self
send
serial
sharp
sharpen
short
shorten
sickle
silk
sing
skunk
skunks
slander
slope
smoke
snot
song
sound
spade
speak
squat
squatter
stampede
stat
steam
steamboat
sterile
sterilize
stock
straight
stripe
subtle
sulphur
super
superbly
swap
tag
tanak
tank
tar
tarp
tea
telephoto
ten
tenth
test
testament
thank
though
thought
tin
toe
tooth
trade
transmigration
transmission
tremble
triffle
trudgeons
trust
trusty
tuesday
turnverein
twice
twist
typhoon
unblendable
uncircumspect
unconventional
understand
up
urban
utility
vector
verdins
vibe
vilified
wall
wander
wash
washed
weak
week
well
will
wiry
wonder
world
worldly
write
wry
yea
yet
you
yuck
*
baber
baben
babe
bobbi
ballloon
baaalllooonnn
disbileaf
dispell
harding
hardeng
hardn
scunk
scianc
qu
quiet
misisp
mcgjwkqsybfv
balan
racan
pop
quiz
zoom
sciser
scisser
scissser
sccisserr
see
for
*
