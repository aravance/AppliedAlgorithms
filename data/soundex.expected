baber:
    bibber
    bobber
baben:
    babboon
babe:
    bob
    bobbie
    bobby
    bop
bobbi:
    bob
    bobbie
    bobby
    bop
ballloon:
    balloon
baaalllooonnn:
    balloon
disbileaf:
    disbelief
    dispel
dispell:
    disbelief
    dispel
harding:
    harden
hardeng:
    harden
hardn:
    harden
scunk:
    science
    skunk
    skunks
scianc:
    science
    skunk
    skunks
qu:
    que
    queue
quiet:
    quit
misisp:
    mississippi
mcgjwkqsybfv:
    mississippi
balan:
    balloon
racan:
    raccoon
    reign
pop:
    pipe
    pop
    pope
quiz:
    quack
    quick
zoom:
    No match found!
sciser:
    scissor
scisser:
    scissor
scissser:
    scissor
sccisserr:
    scissor
see:
    say
    sea
    see
for:
    far
    for
    fore
    four
